src/main/java/ay2021s1_cs2103_w16_3/finesse/logic/commands/ListTransactionCommand.java

```diff
@@ -0,0 +1,26 @@
+package ay2021s1_cs2103_w16_3.finesse.logic.commands;
+
+import static ay2021s1_cs2103_w16_3.finesse.model.Model.PREDICATE_SHOW_ALL_TRANSACTIONS;
+import static java.util.Objects.requireNonNull;
+
+import ay2021s1_cs2103_w16_3.finesse.model.Model;
+import ay2021s1_cs2103_w16_3.finesse.ui.UiState.Tab;
+
+/**
+ * Lists all transactions in the finance tracker to the user.
+ */
+public class ListTransactionCommand extends Command {
```

Comment:
Should this be something like a `ListOverviewCommand` instead, to highlight that it is for the overview tab?

Reply:
Hmm, I went with `ListTransactionCommand` as the commands shouldn't have to care about the state of the UI; that's the job of the `FinanceTrackerParser`.

Reply:
Ok, makes sense.